packages/opencensus-web-instrumentation-zone/test/test-on-page-interaction.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { RootSpan, Tracer } from '@opencensus/web-core';
+import { OnPageInteractionStopwatch } from '../src/on-page-interaction';
+
+describe('OnPageInteraction', () => {
```

Comment:
Should this be 'OnPageInteractionStopwatch'? And should the file be something like `test-on-page-interaction-stopwatch.ts`?

Reply:
Updated.